USER_GUIDE.md

```diff
@@ -41,6 +41,7 @@ Option | Default Value | Valid Options |  Environment Variable | Description
 --subject | This is an email containing your dashboard report | - | OPENSEARCH_SUBJECT | subject for the email
 --note | Hi,\nHere is the latest report! | string or path to text file | OPENSEARCH_EMAIL_NOTE | The email body
 --selfsignedcerts | false | true, false | - | enable or disable self-signed certicates for smtp transport
+--timeout | 300000ms | - | OPENSEARCH_TIMEOUT | timeout for generating report in ms
```

Comment:
the value shouldn't contain `ms` or does it not matter?

Reply:
Good catch. the value shouldn't contain `ms`.